ts/client/src/clientIxParamBuilder.ts

```diff
@@ -2,6 +2,66 @@ import { BN } from '@coral-xyz/anchor';
 import { PublicKey } from '@solana/web3.js';
 import { InterestRateParams, OracleConfigParams } from './types';
 
+export interface TokenRegisterParams {
+  oracleConfig: OracleConfigParams;
+  groupInsuranceFund: boolean;
+  interestRateParams: InterestRateParams;
+  loanFeeRate: number;
+  loanOriginationFeeRate: number;
+  maintAssetWeight: number;
+  initAssetWeight: number;
+  maintLiabWeight: number;
+  initLiabWeight: number;
+  liquidationFee: number;
+  stablePriceDelayIntervalSeconds: number;
+  stablePriceDelayGrowthLimit: number;
+  stablePriceGrowthLimit: number;
+  minVaultToDepositsRatio: number;
+  netBorrowLimitPerWindowQuote: number;
+  netBorrowLimitWindowSizeTs: number;
+  borrowWeightScaleStartQuote: number;
+  depositWeightScaleStartQuote: number;
+  reduceOnly: number;
+  tokenConditionalSwapTakerFeeRate: number;
+  tokenConditionalSwapMakerFeeRate: number;
+  flashLoanSwapFeeRate: number;
+}
+
+export const DefaultTokenRegisterParams: TokenRegisterParams = {
+  oracleConfig: {
+    confFilter: 0,
+    maxStalenessSlots: null,
+  },
+  groupInsuranceFund: false,
+  interestRateParams: {
+    util0: 0.5,
+    rate0: 0.072,
+    util1: 0.8,
+    rate1: 0.2,
+    maxRate: 2,
+    adjustmentFactor: 0.004,
+  },
+  loanFeeRate: 0.0005,
+  loanOriginationFeeRate: 0.005,
+  maintAssetWeight: 0,
+  initAssetWeight: 0,
+  maintLiabWeight: 1.4,
+  initLiabWeight: 1.8,
+  liquidationFee: 0.2,
+  stablePriceDelayIntervalSeconds: 60 * 60,
+  stablePriceDelayGrowthLimit: 0.06,
+  stablePriceGrowthLimit: 0.0003,
+  minVaultToDepositsRatio: 0.2,
+  netBorrowLimitPerWindowQuote: 250_000_000_000,
```

Comment:
Params below look very generous, and others look very conservative, should we revise?

```
  netBorrowLimitPerWindowQuote: 250_000_000_000,
  borrowWeightScaleStartQuote: 5_000_000_000,
  depositWeightScaleStartQuote: 5_000_000_000,
```

Reply:
I copied over the values from register_trustless. Could drop the new borrow limit per day to $5k by default in both?

Reply:
yes, please change borrow limit to $5k